.github/ISSUE_TEMPLATE/new-team-member.md

```diff
@@ -0,0 +1,41 @@
+---
+name: "🙌 Onboard a team member"
+about: Steps to follow to bring on a new team member
+labels: "type: onboard"
+title: "Onboarding <name>"
+---
+
+Onboarding champion: `@<INSERT HANDLE>`
+
+# Onboarding checklist
+
+Follow the checklist below to onboard a new team member!
+
+**Communication**
+
+- [ ] Send them a link to the [Getting started docs](https://team-compass.2i2c.org/en/latest/get-started.html)
+- [ ] Schedule an onboarding meeting
+
+**Accounts**
+
+- [ ] Make email account
```

Comment:
```suggestion
- [ ] Make email account
- [ ] Give access to the 2i2c Google Drive folder
```

Reply:
Ah I think that the reason that access had to be explicitly given was because the folder was under a non `@2i2c.org` account - I've moved that to the `@2i2c.org` account, which means that anybody w/ an `@2i2c.org` address should now be able to access it.

Reply:
That's super!! Thanks!